setup.py

```diff
@@ -292,7 +292,7 @@ def get_dynamic_setup_params():
     # Ansible will also make use of a system copy of python-six and
     # python-selectors2 if installed but use a Bundled copy if it's not.
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*',
-    package_dir={'': 'lib',
+    package_dir={'ansible': 'lib/ansible',
```

Comment:
According to https://setuptools.readthedocs.io/en/latest/setuptools.html#using-a-src-layout, the old version should be enough... Does it cause problems for you?
```suggestion
    package_dir={'': 'lib',
```

Reply:
```
[pts/33@peru /srv/ansible/abadger8]$ python3 setup.py sdist                                                                                                                                                    *[rename-tarball]  (10:17:22)
running sdist
running check
error: package directory 'lib/ansible/ansible' does not exist
```

So yeah, that's why I had to change that line.

Reply:
Hm.. This looks wrong

Reply:
@abadger I just checked locally and cannot reproduce your error. Besides, it's recommended to run `twine check` and not `setup.py check`.

Reply:
I don't know what to tell you....  You started with this PR and then you changed package_dir back to `{'': 'lib',`  ?

Reply:
yep

Reply:
webknjaz gave me a video and I figured out what I was testing wrong... I changed the key back but not the value.   Pushed a fix now.